src/ext/ddtrace.h

```diff
@@ -107,21 +107,21 @@ ZEND_END_MODULE_GLOBALS(ddtrace)
  */
 #if PHP_VERSION_ID < 70000
 #define DDTRACE_ARG_INFO_SIZE(arg_info) ((zend_uint)(sizeof(arg_info) / sizeof(struct _zend_arg_info) - 1))
-#elif PHP_VERSION_ID < 80000
+#elif PHP_VERSION_ID < 90000
```

Comment:
Maybe `80100` instead?
```suggestion
#elif PHP_VERSION_ID < 80100
```